src/coremods/core_channel/cmd_invite.cpp

```diff
@@ -51,7 +51,15 @@ CmdResult CommandInvite::Handle(User* user, const Params& parameters)
 		if (parameters.size() >= 3)
 		{
 			if (IS_LOCAL(user))
-				timeout = ServerInstance->Time() + InspIRCd::Duration(parameters[2]);
+			{
+				unsigned long duration;
+				if (!InspIRCd::Duration(parameters[2], duration))
+				{
+					user->WriteNotice("*** Invalid duration");
```

Comment:
This isn't a very descriptive response. Can you change this (and other places you've used this message) with something that tells the user what specifically it is a response to.

e.g. 

```
** INVITE: expiry duration is invalid

Reply:
I probably won't do `*** THING: whatever` format specifically as that is too close to a snotice, but yep will do

Reply:
This format is used in other places (grep for `\*\*\* [A-Z]+:`) but sure.